### ISSUE TITLE
[Clear Sky] R4 + HDAO Ultra -> crash at start

### ISSUE THREAD

**Issue by donizettilorenzo** [OpenXRay/xray-16#829]
As the title says.
Log:

```
CreateShader: e:\stalker - clear sky\gamedata\shaders\r3\accum_sun.vs is missing. Replacing it with stub_notransform_2uv.vs
CreateShader: e:\stalker - clear sky\gamedata\shaders\r3\ssao_hdao.cs is missing. Replacing it with stub_default.cs
 
FATAL ERROR
 
[error] Expression    : file
[error] Function      : CResourceManager::CreateShader
[error] File          : C:\projects\xray-16\src\Layers\xrRender\ShaderResourceTraits.h
[error] Line          : 623
[error] Description   : Shader file doesnt exist
[error] Arguments     : e:\stalker - clear sky\gamedata\shaders\r3\stub_default.cs
```

High -> working

> CreateShader: e:\stalker - clear sky\gamedata\shaders\r3\accum_sun.vs is missing. Replacing it with stub_notransform_2uv.vs
> Starting engine...

**Comment by Xottab-DUTY** [comment]
Minor bug, HDAO Ultra requires new shaders from COP.

The fix is just hiding the option if new shaders are missing.

**Comment by donizettilorenzo** [comment]
Can you provide the shaders file names in CoP DB files?

**Comment by Xottab-DUTY** [comment]
For sure, this files:
```
ssao_hdao.cs
ssao_hdao_msaa.cs
ssao_hdao_new.ps
```
But, maybe you will also need all **AO related files:
![image](https://user-images.githubusercontent.com/2989212/128087011-40e3e48d-29c1-4589-90b3-b3764667f118.png)

**Comment by donizettilorenzo** [comment]
Thanks!

**Comment by Xottab-DUTY** [comment]
There was a post fix: 7297d346fb36c12d3148289b5cc2d2366107379e
